### PR TITLE
feat: create waitlist page for superadmin user - team starlight

### PR DESCRIPTION
**Pull request by Pharm-ack**
# Description

# Changes proposed

## What were you told to do?
- Create a new waitlist page on the superadmin
- Create the superadmin waitlist page
- Ensure the page is responsive and work on all devices
- Ensure the page works as intended

## What did you do?

- Create a new route in the superadmin dashboard at dashboard/admin/waitlist
- Create the UI of the page according to the design provided
- Ensure the page was responsive across multiple device screen sizes
- Ensure the page work as intended

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
Annotation 2024-07-29 120720

### CODE REVIEW

File: src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+const WaitListTableBody = () => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const handleOpenDialog = () => setIsDialogOpen(true);
+  const handleCloseDialog = () => setIsDialogOpen(false);
+
+  return (
+    <>
+      <tbody className="user-table z-10">
+        {dummyUsers.map((data, index) => {
```
Comment (prudentbird) [inline src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx:25]: Dummy data 🙂

**Review by prudentbird — CHANGES_REQUESTED** [review]

File: src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx (hunk 1 above)
Comment (elitenoire) [reply src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx:25]: allow  🙂

Comment (Pharm-ack) [reply src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx:25]: backend no gree send endpoint........so i was like i will push this first when the endpoint is ready i willl update it!

Comment (prudentbird) [reply src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx:25]: Okay, make a follow up PR

Comment (Pharm-ack) [reply src/app/dashboard/(admin)/admin/(overview)/waitlist-page/_components/waitListTableBody.tsx:25]: sure brother. my backend are on it

**Review by prudentbird — APPROVED** [review]

**Pull request closed without merging**